docker/compose.yaml

```diff
@@ -16,16 +16,57 @@ x-environment: &environment
     __SUPERBLOCKS_AGENT_SERVER_URL: ${__SUPERBLOCKS_AGENT_SERVER_URL:-https://app.superblocks.com}
 
 services:
+  proxy:
+    # Configurable variables for traefik to set up https
+    # SUPERBLOCKS_PROXY_ADMIN_EMAIL
```

Comment:
```suggestion
    # SUPERBLOCKS_LETSENCRYPT_EMAIL
```

Reply:
I have seen in some other software that supports letsencrypt that they do no ask for this and instead they give some default email. I don't think letsencrypt has ever used those emails for anything.